distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -396,3 +420,18 @@ type Any
              (+1 >> *2) 2
     >> : (Any -> Any) -> (Any -> Any) -> Any -> Any
     >> self ~that = x -> that (self x)
+
+
+## PRIVATE
+   Checks if both comparators of the given objects are both of same type and ordered.
+   If they are not of same type, a `Type_Error` is thrown.
+   If the comparators are either `Incomparable`, or unordered, `False` is returned.
```

Comment:
Sorry I only just noticed, but I see some discrepancy here - I don't think that `False` is ever returned here, right? The doc seems to have gotten out of sync - it should say that `Incomparable_Values` is raised.

Reply:
Right. Thanks for the notice. I fixed that in my other PR.

Reply:
Perfect, thanks